cli/tests/fixtures/extensions/permissions/correct-run-perms/main.ts

```diff
@@ -1,10 +1,16 @@
-let cmd = Deno.run({
-  cmd: ['cargo', '--list'],
-  stdout: 'piped'
-})
+import { PhylumApi } from 'phylum';
 
-await cmd.status()
+let cmd = PhylumApi.runSandboxed({
+  cmd: '/bin/echo',
+  args: ['hello'],
+  stdout: 'piped',
+  stderr: 'piped',
+  exceptions: {
+    read: false,
+    write: false,
+    net: false,
+    run: false,
+  }
```

Comment:
They all default to `false` anyway, right? Are these mandatory? Would be nice to just omit them.

Reply:
Those _should_ default to `false` but I need to check again -- and if they don't already, I'd say defaulting to false is the correct expected behavior.